docs/blog/better-ux-with-custom-labels.md

```diff
@@ -134,7 +138,7 @@ const Post = {
     // and the rest of the fields too
   },
   labelResolver: async item => {
-    const { data } = await keystone.executeGraphQL({
+    const { data } = await Keystone.executeGraphQL({
```

Comment:
@JedWatson  this conflicts with what we do in all our docsssss!

Reply:
haha whoops, that got caught in the search/replace for the case change. Wanna change it back?